orga/tests/acceptance/sco-organization-participant-list_test.js

```diff
@@ -276,9 +276,9 @@ module('Acceptance | Sco Organization Participant List', function (hooks) {
             await clickByName('Ajouter l’identifiant');
 
             // then
-            assert.contains('Médiacentre');
-            assert.contains('Identifiant');
-            assert.contains('Nouveau mot de passe à usage unique');
+            assert.ok(screen.getByText('Médiacentre'));
+            assert.ok(screen.getByLabelText('Identifiant'));
+            assert.ok(screen.getByText('Nouveau mot de passe à usage unique'));
```

Comment:
Pareil ici ^^ (je poursuis pas plus la lecture, je viens seulement de voir le tag dev in progress 😅)

```suggestion
            assert.ok(screen.getByRole('textbox', { name: 'Nouveau mot de passe à usage unique' }));
```